napari_btrack/track.py

```diff
@@ -1,24 +1,144 @@
-from pathlib import Path
-from typing import Optional
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Tuple, Union
 
 import btrack
 import napari
+import numpy as np
+import numpy.typing as npt
+from btrack import datasets
+from btrack.config import (
+    HypothesisModel,
+    MotionModel,
+    TrackerConfig,
+    load_config,
+    save_config,
+)
 from btrack.utils import segmentation_to_objects
-from magicgui import magicgui
-from magicgui.widgets import FunctionGui
+from magicgui.application import use_app
+from magicgui.types import FileDialogMode
+from magicgui.widgets import Container, PushButton, Widget, create_widget
+from pydantic import BaseModel
+from PyQt5.QtWidgets import QScrollArea
 
+default_cell_config = load_config(datasets.cell_config())
 
-def run_tracker(objects, config_file_path):
+# widgets for which the default widget type is incorrect
+hidden_variable_names = [
```

Comment:
It's general practice in Python to use all upper case (ie. `HIDDEN_VARIABLE_NAMES`) for variables at the top of a module that are intended to be immutable.